apps/02RockPaperScissors.js

```diff
@@ -9,15 +9,40 @@ var rl = readline.createInterface({
 
 
 function rockPaperScissors(hand1, hand2) {
+  hand1 = hand1.toLowerCase();
+  hand2 = hand2.toLowerCase();
 
   // Write code here
+  if (hand1 === hand2) {
```

Comment:
good spacing here makes it easy to follow your entire if/else stream with all of your conditions